### ISSUE TITLE
Cannot find name 'IResponse'. on TypeScript compile

### ISSUE THREAD

**Issue by toshimaru** [nulab/backlog-js#7]
I'm trying to use backlog-js in TypeScript project.

When I run `tsc`, I get an error:

```
node_modules/backlog-js/dist/backlog.d.ts:20:19 - error TS2304: Cannot find name 'IResponse'.

20       }): Promise<IResponse>;
```

How can I resolve this issue?

**Comment by vvatanabe** [comment]
@toshimaru I' not sure what the cause is. I will chk it.

**Comment by toshimaru** [comment]
I’ve found what causes the issue. There are two main reasons:

1. I was using TypeScript 4
2. I was using `@types/isomorphic-fetch` (which removes `IResponse` interface)

Related PR: #3 (This PR implements `IResponse`)

**Comment by vvatanabe** [comment]
@toshimaru Thanks for investigating into the issue!

**Comment by toshimaru** [comment]
This issue has been fixed on v0.10.0! 👍
